fsevents.js

```diff
@@ -6,6 +6,13 @@
 /* jshint node:true */
 'use strict';
 
+if (process.platform !== 'darwin') {
+  var err = new Error('Cannot find module \'fsevents\'');
```

Comment:
I will throw to
```javascript
if (process.platform !== 'darwin')
  throw new Error('Module \'fsevents\' is not compatible with platform \'' + process.platform + '\'');
```

To @bnoordhuis point, the `scripts.prepublish` entry can probably be left alone. If someone tries a local npm install on a non-darwin machine, it will just fail, as it's just not supported. We should assume someone who's attempting to use `fsevents` has some level of understanding as to what they're doing.

Unfortunately, it appears `spawn` on 0.10 does not support the `stdio: "inherit"` option. I'm still researching someway to get that working.

>...and demonstrate the change will work as intended on all the major platforms.

You will need to better clarify this, as I can't discern from that statement alone what you require to meet that condition. I do believe demonstrating things still work on a Mac, and on at least one non-darwin machine the install becomes benign, would be enough? To that end, can you be more explicit with what you require on this last point?